spec/lib/dfe_data_tables/categories_loader_spec.rb

```diff
@@ -6,12 +6,14 @@
   let(:table_path) { 'spec/fixtures/files/categories_table.xlsx' }
 
   it 'Will perform under a given time' do
-    expect {
-      DfEDataTables::CategoriesLoader.new(table_path)
-    }.to perform_under(10000).ms.sample(10)
+    expect { DfEDataTables::CategoriesLoader.new(table_path) }
+      .to perform_under(5000).ms
```

Comment:
Did you intend to reduce the time to 5s? This looks like it reverts https://github.com/DFE-Digital/npd-find-and-explore/pull/88/files.

Reply:
It would be nice to reduce it, but no, this is an issue with rebasing. I will fix it.